python/interpret-core/tests/utils/test_clean_simple.py

```diff
@@ -23,3 +23,96 @@ def test_clean_dimensions_2d():
     assert init_score[2, 1] == 6
     assert init_score[3, 0] == 7
     assert init_score[3, 1] == 8
+
```

Comment:
Let's eliminate all of these tests and replace them with a test of our public shap interface